src/dawn/engine_dawn.h

```diff
@@ -84,9 +84,8 @@ class EngineDawn : public Engine {
   // pipelines.
   std::unordered_map<amber::Pipeline*, ::amber::dawn::Pipeline> pipeline_map_;
 
-  // TODO(dneto): Remove this. Shaders are attached to the pipeline.
   std::unordered_map<ShaderType, ::dawn::ShaderModule, CastHash<ShaderType>>
```

Comment:
This variable should be deleted now.  You've absorbed it into one of the methods.